frontend/src/pages/CompareV2.tsx

```diff
@@ -306,7 +308,7 @@ function CompareV2(props: CompareV2Props) {
     isError: isErrorMlmdPackages,
     error: errorMlmdPackages,
   } = useQuery<MlmdPackage[], Error>(
-    ['run_artifacts', { runIds }],
+    ['run_artifacts', { runs }],
```

Comment:
The `mlmdPackages` are refetched whenever the `runs` change, not when the `runIds` change, as the page "Refresh" button may cause the `runs` to fetch new artifact information while the `runIds` stay constant.